### PR TITLE
feat: postinstall message cli

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #2200

### Description:
Add postinstall success message 

### Preview:
Screenshot from 2022-05-24 11-31-45

### Scope of the changes (select all that apply):
- [ ] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD 
- [x] CLI

### CODE REVIEW

**Comment by rephus** [comment]
@owlas Could you confirm after publishing and installing from NPM  that it works properly and we can use chalk on the JS file ?

File: packages/cli/src/dbt/context.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import * as path from 'path';
 
 type GetDbtContextArgs = {
     projectDir: string;
+    mainProjectDir?: string;
```
Comment (ZeRego) [inline packages/cli/src/dbt/context.ts:8]: These changes belong to a separate PR that is already open. Should we remove them from this PR ? Otherwise the PR looks good 👍

**Comment by owlas** [comment]
@rephus to publish and check update `package.json` to `version-alpha.0` or something

Then do:
```
rm -rf node_modules
yarn install
yarn build
yarn publish

# In a directory outside the project
npm i -g @lightdash/cli@version-alpha.0
```

That's basically what happens during release, so if it works you should be good

**Comment by rephus** [comment]
I was testing with npm on docker , and it was working fine

```
npm --version
6.14.16
```

But they removed any outputs on `npm@7.0.0` , the script still runs but I can't see anything

https://github.com/npm/cli/issues/3647

**Comment by rephus** [comment]
Closing this PR, this can't be done using `postinstall` on `npm` (See comment above)
